### ISSUE TITLE
build.zig API changes for 0.13.0-dev.211+6a65561e3

### ISSUE THREAD

**Issue by p7r0x7** [00JCIV00/cova#55]
Zig master no longer supports this syntax and requires b.path("path/relative/to/build.zig"), which returns a std.Build.LazyPath.

```
/Users/p7r0x7/.cache/zig/p/1220b4820dfc4510ad9018d4e9f0dabd1f5b4d90162dd32d213d655514c06d61347f/build.zig:14:33: error: no field named 'path' in union 'Build.LazyPath'
        .root_source_file = .{ .path = "src/cova.zig" },
                                ^~~~
/Users/p7r0x7/zig/lib/std/Build.zig:2133:22: note: union declared here
pub const LazyPath = union(enum) {
                     ^~~~~
```

**Comment by 00JCIV00** [comment]
Thanks for raising this. The fix for this is done, I just need to push it up from my machine. 

Of note, I believe there are other changes to Zig v0.13 that won't be fixed until post Cova v0.10 release.

**Comment by 00JCIV00** [comment]
Closed via b46e22d.
